public/javascripts/application.js

```diff
@@ -1,2 +1,28 @@
 // Javascript Goes Here~
 //
+
+$(document).ready(function() {
+
+  var Number = (function(){
+
+    // This is a private variable and cannot be called outside the function.
+    var start = 0;
+    console.log(start);
+
+    return {
+
+      // These are public functions calling the private variable and return the start value.
+      add: function(){
```

Comment:
Please move these functions outside of the return clause and then return the var you set them to in the return object.  I dont think it changes the behavior, but it makes it a bit clearer.